Hymns/Display/SoundCloudView.swift

```diff
@@ -0,0 +1,44 @@
+import SwiftUI
+
+struct SoundCloudView: View {
+    @State private var isLoading = true
+    @Binding var showSoundCloud: Bool
+    @Binding var soundCloudinitiated: Bool
+    var searchTitle: String
+
+    var body: some View {
+        Group<AnyView> {
+            guard let url =
+                "https://soundcloud.com/search?q=\(self.searchTitle)".toEncodedUrl else {
+                    return ErrorView().eraseToAnyView()
+            }
+            return VStack {
+                HStack(spacing: 10) {
+                    Button(action: {
+                        self.showSoundCloud = false
+                        self.soundCloudinitiated = false
+                    }, label: {
+                        Image(systemName: "xmark.square.fill").accentColor(.red)
+                    })
+                    Button(action: {
+                        self.showSoundCloud.toggle()
+                    }, label: {
+                        Image(systemName: "minus.square.fill").accentColor(.accentColor)
+                    })
+                    Spacer()
+                }.padding()
+                ZStack {
+                    SoundCloudWebView(isLoading: self.$isLoading, url: url).eraseToAnyView().opacity(isLoading ? 0 : 1)
+                    ActivityIndicator().maxSize().eraseToAnyView().opacity(isLoading ? 1 : 0)
```

Comment:
What if you didn't have this? I see like 2-3 activity indicators when trying to load the webview, which is a bit weird. It's not a blocker, but can we reduce this to only show 1 indicator?

Reply:
I reduced this one here. That brings us from 3 to 2. I'm not sure if I can reduce another. I'll try

Reply:
I think it's because one is coming from the webKit and the other is from the soundcloud site itself

Reply:
Oh, is the `webKit` one also not controlled by us?